cms/src/components/content/ContentOverview.vue

```diff
@@ -77,6 +79,44 @@ if (!Object.entries(TagType).some((t) => t[1] == tagTypeString)) tagTypeString =
 
 const titleType = tagTypeString ? tagTypeString : props.docType;
 router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} overview`;
+
+const searchTerm = ref("");
+
+const sortOptionsAsRef = ref(null);
+
+const showSortOptions = ref(false);
+
+const selectedSortOption = ref("updatedTimeUtc");
+
+watch(searchTerm, () => {
+    setTimeout(() => {
+        queryOptions.value.search = searchTerm.value;
+    }, 500);
+});
+
+watch(selectedSortOption, () => {
+    queryOptions.value.orderBy = selectedSortOption.value as
+        | "title"
+        | "updatedTimeUtc"
+        | "publishDate"
+        | "expiryDate";
```

Comment:
Same comment as the other PR on the cast here, don't think it's needed.

Reply:
Gave an error on my side. this fixed the error